client/my-sites/plans/current-plan/current-plan-thank-you/backup-thank-you.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { localize } from 'i18n-calypso';
+import React, { Fragment } from 'react';
```

Comment:
Nit: It's common to keep the `React` import as the first one. Historically it has helped us when running codemods.